monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -7,6 +7,8 @@
 import arrow
 import datetime
 from datetime import datetime, timedelta
+from monitoring.rid_qualifier.utils import QueryBoundingBox, FlightPoint, GridCellFlight
```

Comment:
All of these classes are redefined below, overriding this import.

Reply:
I think this has to with the merging of the branches, I have removed the redefined ones and updated the ones in `utils.py`